database/src/main/java/org/akvo/flow/database/britedb/BriteSurveyDbAdapter.java

```diff
@@ -565,6 +565,9 @@ public void updateSurvey(ContentValues updatedValues, String surveyId) {
     public void deleteAllSurveys() {
         briteDatabase.delete(Tables.SURVEY, null);
         briteDatabase.delete(Tables.SURVEY_GROUP, null);
+        briteDatabase.delete(QuestionGroupTable.TABLE_NAME, null);
+        briteDatabase.delete(FormUpdateNotifiedTable.TABLE_NAME, null);
+        briteDatabase.delete(LanguageTable.TABLE_NAME, null);
```

Comment:
Any reason not to follow the same naming convention as the previous tables. i.e. `Tables.SURVEY`?

Reply:
the new tables I have been adding, have code inside their own class with table name and columns and create code. I find it more clear than having Tables just the list of tables and then columns somewhere else. Also the DatabaseHelper becomes really large with lots of code so I separated it this way